### DISCUSSION TITLE
Packagist missing versions

### DISCUSSION THREAD

**Discussion by duncancumming** [getsentry/sentry-php#1297] [Q&A] [answered]
Trying to install sentry/sdk using Composer, kept getting this error

`- The requested package sentry/sdk 1.11.0 exists as sentry/sdk[0.0.2, 2.0.0, 2.0.1, 2.0.2, 2.0.3, 2.0.4, 2.1.0, 2.2.0, 3.0.0, 3.0.0-beta1, 3.1.0, 3.1.1, dev-master] but these are rejected by your constraint.`

Checking Packagist:
https://packagist.org/packages/sentry/sdk

It only has those releases listed, not any others.  I realise there's workarounds here.  But given Packagist is the default repository used by Composer, it would make sense to have the other releases listed here too.

**Comment by stayallive** [comment, marked as answer]
You are looking at a meta package which you should install the latest version of. This is a convenience package that includes the recommended HTTP client.  

If you want a specific version of the SDK itself you need this package: https://packagist.org/packages/sentry/sentry. 

You can also require both packages to select a specific SDK while also pulling in the recommended HTTP client. 

It seems like you are trying to install version 1.11? This is a really old version, are you sure that is the one you need? That version predates the metapackage anyway 😅

**Reply by duncancumming** [reply]
Apologies, confusion on my part between get sentry/sentry and sentry/sdk

**Reply by stayallive** [reply]
No worries, it happens! All sorted then?
